### ISSUE TITLE
Clicking on disclosed subfolders does nothing

### ISSUE THREAD

**Issue by michaelroper** [nystudio107/disclosedassets#1]
Hey Andrew - I've installed this plugin, and have noticed that while it is indeed disclosing all the asset subfolders, clicking on the subfolders actually seems to do nothing, unless I click on the arrow to hide them and then again to show them. Can't see any errors in the dev tools console either... ¯\_(ツ)_/¯

**Comment by sergeifilippov** [comment]
Confirmed.

Also tried the suggested js code in admin:

```js
$(document).ready(function() {
  $('div .toggle').parent('li').addClass('expanded');
});
```

Here's a gif: 
![expand-not](https://user-images.githubusercontent.com/123676/30521190-d64bcc0e-9c0f-11e7-846b-62f847de592c.gif)

## System
- Craft CMS 2.6.2990
- nginx 1.12
- php7.1

## Tested on
- macOS Sierra 10.12.6
- Chrome Version **61.0.3163.91** (Official Build) (64-bit)
- Firefox **55.0.3**
- Safari **10.1.2 (12603.3.8)**

**Comment by khalwat** [comment]
Well, that's disappointing. :) I have an idea of how to fix it tho.

**Comment by khalwat** [comment]
This is fixed in 1.0.1:

https://github.com/nystudio107/disclosedassets/releases/tag/1.0.1

An unfortunate side-effect is that it no longer auto-discloses asset sub-folders in the "choose an asset" modal dialog.

To address that, there would have to be some kind of event we listen for when a modal pops up, and then we can simulate a click on the toggles.
